.gitignore

```
@@ -81,8 +81,8 @@ libtool
 ltmain.sh
 cppclient/watchmanclient.pc
 /website/.jekyll-metadata
-/rust/**/Cargo.lock
-/rust/**/target/
+/**/rust/**/Cargo.lock
+/**/rust/**/target/
```

(copying for @jdelliot from Phabricator)

Can we just use this:
```suggestion
rust/**/Cargo.lock
rust/**/target/
```

---

That does not work for me.  I see all the build artifacts included when that happens.

---

You can add a `.gitignore` inside `rust/`, so the rules can be `Cargo.lock` and `target` there, which is cheaper to handle.

---

that is awesome!  I had no idea you could used nested .gitignores.  I have put it in the `rust` subdirectory instead of putting one in rust/watchman_client and one in rust/serde